Chapter 1 and 2 corrections (#99)
chhoumann/p8: report/src/problem_analysis/hci/test_results.tex

One of the problems described in Section \ref{sec:understanding_the_problem} explains that it is tedious to modify ongoing meetings. 
During the test, it was found that the prototype was received positively in this regard. 
Users found the interaction with and information on the prototype intuitive. 

> They understood the different situations depicted on the prototype and understood how to interact with the prototype to complete the test tasks described in Appendix \ref{appendix:usability_tasks} without issues.

Reviewer: ```suggestion
They understood the different situations depicted in the prototype as well as how to interact with the prototype to complete the test tasks described in Appendix \ref{appendix:usability_tasks} without issues.
```

During the debriefing, it was mentioned that the addition of being able to extend and modify meetings using a tablet would be an improvement to their current workflows.
However, it was speculated whether other approaches to making these changes to meetings could be done. 

> The prototype focuses on the integration with existing systems already in use by the stakeholder and relies on user interaction for managing the meetings.

Reviewer: ```suggestion
The prototype focuses on the integration with existing systems already in use by the stakeholder, and it depends on user interaction for managing the meetings.
```

> Since both the prototype and their current system is fully dependent on users correctly managing the meetings, some aspects of the problematic situation cannot be solved by this product. 

Reviewer: ```suggestion
Since both our prototype and their current system fully depend on users correctly managing the meetings, some problematic situations cannot be solved by this prototype. 
```

For instance, ending meetings early still relies on users remembering to register this is the system. 

> Therefore, a proper solution should implement functionality that is able to automize necessary aspect of meeting management, minimizing reliance on users.

Reviewer: ```suggestion
Therefore, a proper solution should implement functionality that is able to automate necessary aspect of meeting management, minimizing reliance on users.
```

> Therefore, a solution had to implement functionality that is able to automize necessary aspect of meeting management, minimizing reliance on users.

Reviewer: ```suggestion
Therefore, a proper solution should implement functionality that is able to automize necessary aspect of meeting management, minimizing reliance on users.
```
